composer.json

```diff
@@ -8,7 +8,6 @@
     "laravel/framework": "5.5.*",
     "laravel/tinker": "^1.0",
     "aws/aws-sdk-php-laravel": "^3.1",
-    "barryvdh/laravel-debugbar": "^3.1.0",
```

Comment:
I thought that this was to add clockwork with debugbar. Are we removing debugbar?

Reply:
Clockwork works with debugbar already (with a limited feature set)!

The downside is that doesn't support a lot of Clockworks most handy features – the ability to work with JSON API responses (which I feel like are most often what we want to inspect) & the `clock()->startEvent()` and `clock()->stopEvent()` methods for measuring performance.

I'd love to try this out for a little bit and see what people think, and we can always switch back to Debugbar if it doesn't seem like an improvement.

Reply:
Ooooo you mean Clockwork contains debugbar? When you said that before I thought you meant they work in conjunction! Cool that sounds good to me!

Reply:
Sorry, that was awkwardly phrased! The [Clockwork chrome extension](https://chrome.google.com/webstore/detail/clockwork/dmggabnehkmmfmdffgajcflpdjlnoemp?hl=en) can function in a "limited" mode while the `barryvdh/laravel-debugbar` extension is installed w/ Composer, since it reads some of the same data. That's what I've been doing for a while now.

Removing the `barryvdh/laravel-debugbar` _will_ remove the debugbar interface, so you'll have to visit the `rogue.dev/__clockwork` page on your local app (or install the Chrome extension) in order to see the same kind of debugging information.

From my time playing with it, I think Clockwork should handle all the exact same things we were using debugbar for, in addition to having a better workflow for inspecting API requests & better tools for measuring performance.

Reply:
Let's just consider this a test drive, and we can always switch back in a few weeks! 🚗

Reply:
Ahhh I get it now! Makes sense and sounds good